scipy/signal/fir_filter_design.py

```diff
@@ -594,3 +596,106 @@ def remez(numtaps, bands, desired, weight=None, Hz=1, type='bandpass',
     bands = np.asarray(bands).copy()
     return sigtools._remez(numtaps, bands, desired, weight, tnum, Hz,
                            maxiter, grid_density)
+
+
+def firls(numtaps, bands, desired, weight=None, fs=2):
```

Comment:
`firwin` and `firwin2` use `nyq=1.0`. Of course, there's no real functional difference, but it would be best to be consistent across FIR filter design functions. (`remez` uses `Hz=1` which is yet another permutation...)